starter/script.js

```diff
@@ -1 +1,119 @@
+  buffer()
+  console.log("What should you do now?\nGive the family a good talking to?\nSabotage the snowmobile and the two way radio?\nPut on some comfy clothes, settle in by the fire, and have boardgame night with the family?");
+  buffer();
+  console.log("Please enter 'sabatoge' 'boardgames' or 'wendy darling'")
+  buffer();
+  var secondInput = prompt("Please enter 'sabatoge' 'boardgames' or 'wendy darling'");
+
+  if (secondInput == "wendy darling") {
+    console.log("Ok, you decided to have a talk with Wendy, the light of your life.");
+    console.log("Be sure to let her know you're not gonna hurt her.");
+    buffer();
+    thirdRound();
+  }
+  else if (secondInput == "sabatoge") {
+    console.log("Ok, you decided to sabotoge the means of outide communication.");
+    console.log("Now there’s no way to contact the outside world! Great?");
+    buffer();
+    thirdRound();
+  }
+  else if (secondInput == "boardgames") {
+    console.log("Ok, you decided to get in some quality family time.");
+    console.log("I'm sure this whole thing will blow over soon.  In the meantime, break out the chutes and ladders!");
+    buffer();
+    thirdRound();
+  }
+}
+buffer();
+
+var thirdRound = function() {
+  console.log("Oh no!  Wendy knocked you out with a baseball bat, dragged you into the kitchen, and locked you inside the pantry.  Luckily, Grady the ghost let you out.  Now it’s time to grab that firefighter’s ax and go teach your family a lesson. What is it Ed Mcmahon says at the beginning of the Tonight Show?");
+  buffer()
+  console.log("Where should you look for your family?\nThe Gold Room, the hedge maze, or Room 237?");
+  buffer();
+  console.log("Please enter either 'gold', 'maze', or '237'")
+  buffer();
+  var thirdInput = prompt("Please enter 'gold' 'maze' or '237'");
+
+  if (thirdInput == "gold") {
+    console.log("Ok, you picked gold.");
+    console.log("Hey! There’s a fancy party going on!  The bartender, Lloyd, slips you a bottle of bourbon, a glass, and a little bit of ice.  Let’s party like it’s 1921!");
+    buffer();
+
+  }
+  else if (thirdInput == "maze") {
+    console.log("Ok, you picked maze.");
+    console.log("Uh oh, it’s a lot colder out here than you thought it would be.  And it’s pretty hard to find your way around with blah blah blah.  You should probably take a rest and wait it out until morning. ");
+    buffer();
+
+  }
+  else if (thirdInput == "237") {
+    console.log("Ok, you picked 237");
+    console.log("Woah, there is some weirdness going on at this hotel.  Was that guy wearing a dog costume?  Well… I guess you should try anything once...");
+    buffer();
+
+  }
+}
+buffer()
+
+var caretaker = function() {
+  console.log("Just out of curiosity, is this your first winter as caretaker here?")
+var fourthInput = function() {
+  prompt("Please enter 'yes' or 'no'")
```

Comment:
probably want to save the value for the `fourthInput` into a variable and return that variable here, then invoke the function in the `while` loop. Or just not define it in the scope of a function.